### PR TITLE
feat: import tiles from v2 documents

### PR DESCRIPTION
**Pull request by kswenson**
Puts in place the infrastructure for tiles to specify their own tile-specific importers. Supported tiles are imported at their correct locations in the document and with their appropriate titles. Currently, the case table is imported with essential fidelity and graphs are imported at appropriate locations but as unconfigured case plots. The existing v2 import code for graphs will need to be modified to work within the new importer infrastructure. Support for other tiles can be added as separate stories.

### CODE REVIEW

**Review by bfinzer — APPROVED** [review]
👍🏻LGTM
Synchronous review was very helpful!

File: v3/src/components/graph/components/graph-component.tsx [hunk 1]
```diff
@@ -38,18 +40,22 @@ export const GraphComponent = observer(({tile}: ITileBaseProps) => {
   const {setNodeRef} = useDroppable({id: dropId})
   setNodeRef(graphRef.current)
 
+  if (!isGraphModel(graphModel)) return null
```
[the remark this answers is not in the document]
Comment (kswenson) [reply v3/src/components/graph/components/graph-component.tsx:43]: Good catch -- will fix.

File: v3/src/components/graph/components/graph-component.tsx [hunk 2]
```diff
@@ -38,18 +40,22 @@ export const GraphComponent = observer(({tile}: ITileBaseProps) => {
...
-          <GraphModelContext.Provider value={graphModel}>
-            <Graph graphController={graphController}
-                   graphRef={graphRef}
-                   showInspector={uiState.isFocusedTile(tile?.id)}
-            />
-          </GraphModelContext.Provider>
-        </AxisLayoutContext.Provider>
-      </GraphLayoutContext.Provider>
-    </InstanceIdContext.Provider>
+    <DataSetContext.Provider value={data}>
```
[the remark this answers is not in the document]
Comment (kswenson) [reply v3/src/components/graph/components/graph-component.tsx:46]: Yes, line 22 prioritizes the model's notion of data set, which can be dynamically assigned (via its link to the appropriate shared model).

File: v3/src/components/graph/hooks/use-graph-controller.ts [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
-  const v2Document = useV2DocumentContext()
+  // const v2Document = useDocumentContext()
   const dataset = useDataSetContext()
 
-  useEffect(() => {
-    v2Document && graphController.processV2Document(v2Document)
-  }, [graphController, v2Document])
+  // useEffect(() => {
+  //   v2Document && graphController.processV2Document(v2Document)
+  // }, [graphController, v2Document])
```
[the remark this answers is not in the document]
Comment (kswenson) [reply v3/src/components/graph/hooks/use-graph-controller.ts:19]: Will do

**Pull request merged**